src/cobra/test/test_util/test_array.py

```diff
@@ -24,7 +25,8 @@ def test_dense_matrix(model):
 
 
 def test_sparse_matrix(model):
-    scipy = pytest.importorskip("scipy")
+    """Test sparse stoichiometric matrix creation."""
+    _ = pytest.importorskip("scipy")
```

Comment:
You can just ignore the return value completely.

```suggestion
    pytest.importorskip("scipy")
```